src/iadmin.cpp

```diff
@@ -2387,6 +2460,24 @@ usage( char *subOpt ) {
         ""
     };
 
+    char* get_delay_server_info_usage[] = {
+        "get_delay_server_info",
+        " ",
+        "Prints information about the delay server as JSON.",
+        " ",
+        "This command allows administrators to identify which server is running the",
+        "delay server and if the delay server is being migrated.",
+        " ",
+        "This information is retrieved from R_GRID_CONFIGURATION.",
```

Comment:
"the R_GRID_CONFIGURATION table."

?

not sure it's necessary, just might be clarifying for a new reader.

Reply:
I actually had that originally lol, but tried to match set_delay_server's help text.

Will change to what you've recommended.

Reply:
nice - right back on me.     i'm sure i went with brevity at the time.   let's use more words.